From: "David S. Miller" <davem@davemloft.net>
Date: Sat, 19 Oct 2013 19:09:18 -0400
Subject: [PATCH 046/181] bonding: Remove __exit tag from bond_netlink_fini().
Patch-mainline: v3.13-rc1
Git-commit: a729e83ad6f6de2cf11c0da4a5a7c0b3924d8335
References: fate#316924 bnc#875631 bnc#876145

It can be called from the module init function, so it cannot
be in the exit section.

Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Michal Kubecek <mkubecek@suse.cz>

---
 drivers/net/bonding/bond_netlink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/bonding/bond_netlink.c b/drivers/net/bonding/bond_netlink.c
index fe3500b..7661261 100644
--- a/drivers/net/bonding/bond_netlink.c
+++ b/drivers/net/bonding/bond_netlink.c
@@ -123,7 +123,7 @@ int __init bond_netlink_init(void)
 	return rtnl_link_register(&bond_link_ops);
 }
 
-void __exit bond_netlink_fini(void)
+void bond_netlink_fini(void)
 {
 	rtnl_link_unregister(&bond_link_ops);
 }
-- 
1.8.1.4

